wordpress-mobile/WordPress-iOS
# Redirect Terms and service to open the page in an external web view

**Fixes** https://github.com/wordpress-mobile/WordPress-iOS/issues/14756

Additional discussion is available in https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/605

## To test:
- Be logged out
- Tap "Sign up for WordPress.com" 
- Tap "Terms of Service"
- Expect to be taken to see the Terms of service page open in your defined browser.

## Regression Notes
1. Potential unintended areas of impact
None that I can think of

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
None

## PR submission checklist:
